SSA/Experimental/IntrinsicAsymptotics.lean

```diff
@@ -720,38 +901,38 @@ theorem denote_splitProgramAt {pos : ℕ} {prog : ICom Γ₁ t}
 `target`. If it can match the variables, it inserts `rhs` into the program
 with the correct assignment of variables, and then replaces occurences
 of the variable at position `pos` in `target` with the output of `rhs`.  -/
```

Comment:
it seems like the docstring doesn't match the function anymore - could we update the docstring?